Projects/App/Sources/RootView.swift

```diff
@@ -7,22 +7,34 @@
 
 import ComposableArchitecture
 import DesignSystem
+import KakaoLogin
 import SwiftUI
 
-public struct RootView: View {
-  public let store: StoreOf<RootCore>
+struct RootView: View {
+  let store: StoreOf<RootCore>
   
-  public init(store: StoreOf<RootCore>) {
+  init(store: StoreOf<RootCore>) {
     self.store = store
   }
   
-  public var body: some View {
-    VStack {
-      LottieView(type: .confetti)
-        .frame(width: 500, height: 500)
+  var body: some View {
+    Group {
+      if !store.isLogin {
+        LoginView(
+          store: store.scope(
+            state: \.login,
+            action: \.login
+          )
+        )
+        .onOpenURL { url in
+          store.send(.onOpenURL(url))
+        }
+      } else {
+        Text("로그인이 되었습니다.")
+      }
     }
-    .onOpenURL { url in
-      store.send(.onOpenURL(url))
+    .onAppear {
+      store.send(.checkAccessToken)
```

Comment:
```suggestion
      store.send(.onAppear)
```
View로부터 받는 액션은 Modifier 혹은 컴포넌트의 이름과 최대한 연관지어 명명하면 Reducer내에서 로직 구조를 이해하기 편리하다는 장점이 있습니다